floss/language/go/extract.py

```diff
@@ -18,6 +18,50 @@
 MIN_STR_LEN = 6
 
 
+def calculate_coverage(enhanced_static_strings: List[StaticString], static_strings: List[StaticString]) -> float:
+    all_static_string_list = []
+
+    for string_obj in static_strings:
+        all_static_string_list.append(string_obj.string)
+
+    extracted_strings = []
+    total_chars_covered = 0
+
+    for j in enhanced_static_strings:
+        k = j.string
+        for string_obj in static_strings:
+            if k in string_obj.string:
+                extracted_strings.append(string_obj.string)
+                total_chars_covered += len(string_obj.string)
```

Comment:
these are not used right now, would be interesting to see stats based on them

Reply:
another approach here may be to construct one large string of all static strings and find the extracted_strings in there once, something like:

```python
    len_all = len(all_ss_strings)
    for rs in recovered_strings:
        if rs in all_ss_strings:
            rs_len += len(rs)
            all_ss_strings = all_ss_strings.replace(rs, "", 1)
    print(f"Percentage of strings extracted: {round(100 * (rs_len / len_all))}%")
```